evaluate.sh

```diff
@@ -89,7 +90,12 @@ if [ ! -d "$HOME/.sdkman/candidates/java/$DEFAULT_JAVA_VERSION" ]; then
   print_and_execute sdk install java $DEFAULT_JAVA_VERSION
 fi
 
-# 4. Install missing SDK java versions in any of the prepare_*.sh scripts for the provided forks
+# 4. make sure the build java version is installed
+if [ ! -d "$HOME/.sdkman/candidates/java/$BUILD_JAVA_VERSION" ]; then
+  print_and_execute sdk install java $BUILD_JAVA_VERSION
+fi
```

Comment:
That should not be needed, as all JDKs referenced by all prepare scripts will be installed already.

Reply:
👍 will remove it

Reply:
Removed in the latest push